### PR TITLE
feat(liveness): remove 3 second countdown from liveness check

### PR DESCRIPTION
**Pull request by thaddmt**
#### Description of changes
- Per UX request, removing the countdown from liveness check
- Aligned with PM and UX on the new experience
  - User will go straight from `move back` to `connecting` to matching the oval
  - For users with longer `connecting` steps we will show an error if they move their face closer during the countdown
 
foobar
timeout

#### Checklist

- [ ] Have read the [Pull Request Guidelines](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md)
- [ ] PR description included
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] `yarn test` passes and tests are updated/added
- [ ] No side effects or [`sideEffects`](https://github.com/aws-amplify/amplify-ui/blob/main/packages/react/CONTRIBUTING.md#code-standards) field updated

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/react-liveness/package.json [hunk 1]
```diff
@@ -60,7 +60,6 @@
     "@xstate/react": "3.0.0",
     "classnames": "2.3.1",
     "nanoid": "3.1.31",
-    "react-countdown-circle-timer": "^2.5.4",
```
Comment (reesscot) [inline packages/react-liveness/package.json:63]: nice!

**Review by reesscot — APPROVED** [review]

**Comment by reesscot** [comment]
Should the error message still refer to the countdown if it's missing? 
image

**Comment by thaddmt** [comment]
> Should the error message still refer to the countdown if it's missing? image

yep! just waiting on new text from the ux

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts [hunk 2]
```diff
@@ -163,7 +163,7 @@ describe('Liveness Machine', () => {
     jest.advanceTimersToNextTimer(); // waitForSessionInformation
     await flushPromises(); // detectFaceDistanceBeforeRecording
     jest.advanceTimersToNextTimer(); // checkFaceDistanceBeforeRecording
-    service.send({ type: 'START_RECORDING' });
+    // await flushPromises();
```
Comment (calebpollman) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts:166]: Assuming this is unintentional? 😅

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/__tests__/index.test.ts:166]: whoops haha, this is when i was testing stuff :P will fix

**Review by calebpollman — DISMISSED** [review]

**Review by reesscot — APPROVED** [review]

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts [hunk 3]
```diff
@@ -739,6 +752,7 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
         context.componentProps!.onError?.(error);
       },
       callErrorCallback: async (context, event) => {
+        console.log({ event });
```
Comment (wlee221) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:755]:
```suggestion
```

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts [hunk 4]
```diff
@@ -198,46 +202,46 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
...
+        initial: 'waitForSessionInfo',
         states: {
-          detectFaceDistanceDuringCountdown: {
+          waitForSessionInfo: {
+            after: {
+              0: {
+                target: '#livenessMachine.recording',
+                cond: 'hasServerSessionInfo',
+              },
+              10: { target: 'detectFaceDistanceDuringLoading' },
```
Comment (wlee221) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:225]: Just for my curiosity, why do we have to do these two tasks async? (0 vs 10). Is it possible to combine them?

Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:224]: I guess this is the pattern I originally saw done in this machine for essentially doing conditionals. If the first condition does not pass then we do the second one 10 ms later. If there is a better way of doing it happy to change it :)

File: packages/react-liveness/package.json (hunk 1 above)
Comment (zchenwei) [inline packages/react-liveness/package.json:63]: I do not see a `yarn.lock` update?

Comment (thaddmt) [reply packages/react-liveness/package.json:63]: oh let me regenerate it! good catch

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts (hunk 4 above)
Comment (wlee221) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:224]: If you're only interested in doing one or the other, you can do this:

```ts
waitForSessionInfo: {
  always: [
    {
      target: '#livenessMachine.recording',
      cond: 'hasServerSessionInfo',
    },
    { target: 'detectFaceDistanceDuringLoading' },
  ],
},
```
and essentially work as `if` `else` statement. This is more [idiomatic](https://xstate.js.org/docs/guides/guards.html#multiple-guards) to handle conditional transitions 🙏

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts [hunk 5]
```diff
@@ -198,46 +201,46 @@ export const livenessMachine = createMachine<LivenessContext, LivenessEvent>(
...
+          checkFaceDistanceDuringLoading: {
             after: {
               0: {
                 target: 'failure',
                 cond: 'hasNotEnoughFaceDistanceBeforeRecording',
               },
-              200: {
-                target: 'detectFaceDistanceDuringCountdown',
+              100: {
+                target: 'waitForSessionInfo',
```
Comment (wlee221) [inline packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:243]: Can we replace this with the above recommendation too?

File: packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts (hunk 4 above)
Comment (thaddmt) [reply packages/react-liveness/src/components/FaceLivenessDetector/service/machine/index.ts:224]: ah thanks will change!

**Review by zchenwei — APPROVED** [review]
🚢

**Review by wlee221 — APPROVED** [review]

**Comment by gustavovalverde** [comment]
Just an FYI as we have users using this functionality, and it's very confusing having the `Avoid moving closer during countdown and ensure only one face is in front of camera.` as there is no countdown, and there are no more faces in front of the camera.

**Comment by reesscot** [comment]
@gustavovalverde that's a good callout, can you please open a github issue for that?

**Comment by thaddmt** [comment]
@gustavovalverde that string should be updated in the 2.x version to be `Avoid moving closer when connecting.` and no longer has references to countdown. On the 1.x version it should still be the same though

**Pull request merged**